src/main/java/com/wojtek/parkingmeter/model/CarEntity.java

```diff
@@ -0,0 +1,65 @@
+package com.wojtek.parkingmeter.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "cars")
+public class CarEntity {
+
+    public CarEntity() {
+    }
+
+    public CarEntity(String nr_plate) {
+
+        this.nr_plate = nr_plate;
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "nr_plate")
+    private String nr_plate;
+
+    @OneToMany(mappedBy = "carEntity", cascade = {CascadeType.ALL})
+    @JsonIgnoreProperties(value = {"carEntity"})
+    private List<TicketEntity> ticketEntities;
```

Comment:
jak ktoś będzie u nas parkował codziennie przez parę lat to zawsze chcemy wyjmować jego wszystkie bilety?

Reply:
No faktycznie, nie, nie chcemy. Pomyślę jak zrobić to lepiej.